examples/chatbot/config.py

```diff
@@ -69,12 +69,14 @@
                 "dataset_preset": search_space.Constant("dstc11"),
                 "model_preset": search_space.Categorical(
                     [
-                        "gpt-3.5-turbo",
+                        # "gpt-3.5-turbo",
                         "gpt2",
                         "gpt2-xl",
                         "llama-7b",
                         "vicuna-7b",
                         "mpt-7b-chat",
+                        "vicuna-7b-v1.3",
```

Comment:
What do you think our norm should be for the default options? e.g. vllm isn't installed by default, should it be in the default config of this example?

Reply:
Yeah, that's a good question. vllm is installed through the requirements file for this example now, but there's also an argument that we should make this configurable. I'll try to think about the best way to do that.

Reply:
I will handle this in a future commit: https://github.com/zeno-ml/zeno-build/issues/160